Earthfile

```diff
@@ -15,17 +38,27 @@ RUN cargo install \
     cargo-chef \
     jaq
 
+RUN cargo install maturin --locked
+
+# Create a virtual environment for the python dependencies.
+RUN python3.9 -m venv .venv
+RUN . .venv/bin/activate && pip install --upgrade pip
+
 # This target prepares the recipe.json file for the build stage.
 chef-prepare:
-    COPY --dir crates Cargo.toml .
+    COPY --dir crates .
+    COPY --dir python .
+    COPY Cargo.toml .
     RUN cargo chef prepare
```

Comment:
Does chef prepare actually use any of the python code to do its work?

Reply:
I don't know. However, not copying the python folder means that the rust workspace cannot be compiled because of missing members